docs/meetings/collab-cafe/2023.md

```diff
@@ -1,5 +1,79 @@
 # 2023 Collaboration Cafe Notes Archive
 
+## 2023-09-19
+
+### AWS/Curvenote MyBinder federation member
+
+- **Description:** New AWS/curvenote mybinder federation member: Steve Purves has donated AWS credits and is keen to become an official member. Feedback welcome on this PR
+    - <https://github.com/jupyterhub/mybinder.org-deploy/pull/2652>
+    - Related discussion/background: <https://github.com/jupyterhub/mybinder.org-deploy/issues/2556>
+
+#### Notes
+
+- Will it work against AWS ECR? Next PR
+    - Deployment of cloud infra is in one PR: <https://github.com/jupyterhub/mybinder.org-deploy/pull/2652>
+    - The BinderHub chart is deployed with extraConfig that defines a new DockerRegistry sub class that takes care of the complexity of working against a AWS ECR container registry (by interacting with a Golang based service Simon has developed).
+        - <https://github.com/manics/binderhub-container-registry-helper/tree/main#api-endpoints>
+    - Curvenote BinderHub chart deployment: <https://github.com/jupyterhub/mybinder.org-deploy/pull/2698>
+
+### Lowest level of maintenance for JupyterHub org project's
+
+- **Description:** In github.com/jupyterhub has a lot of projects, if possible I'd (@consideRatio) love us to manage to ensure a some basic level of maintenance in all of the projects - or, deprecate/archive them etc.
+
+#### Notes
+
+- We have lots of repos, concerns about level of support each repo gets.
+- Would like some clear expectations about what level of maintenance we are promising for being an "official jupyterhub repo"
+- Sarah is in touch with an experienced leader (missed name) might be available to for discussion on open source leadership. Sarah is meeting next week.
```

Comment:
@sgibson91 Any chance you could fill in the missing name (assuming it's fine for it to be public? otherwise remove it). Then go ahead and merge!